modules/apps/frontend-js/frontend-js-module-launcher/src/main/resources/META-INF/resources/webpack_federation.js

```diff
@@ -83,6 +84,20 @@
 		}
 	}
 
+	/**
+	 * Transform a webpack chunk script file name.
+	 *
+	 * The webpack federation runtimes for projects built with npm-scripts
+	 * delegate to this method the logic to compose the URL to retrieve the
+	 * chunk files from server.
+	 */
+	function transformChunkScriptFilename(chunkScriptFileName) {
+		return (
+			chunkScriptFileName + '?languageId=' +
+				Liferay.ThemeDisplay.getLanguageId()
```

Comment:
While we're at it... do we have caching concerns here? I don't see the `t?` param that we add in other places... maybe this is no longer necessary since different builds might have different hashes generated?

![just-sayin](https://user-images.githubusercontent.com/905006/110506447-01c69100-80f7-11eb-86fe-c058feccd7f3.gif)

Reply:
This :point_right:  https://issues.liferay.com/browse/LPS-128889